src/test/java/uk/gov/digital/ho/pttg/application/namematching/candidates/NamesWithAbbreviationSpaceCombinationsAbbreviatedNamesFunctionsTest.java

```diff
@@ -32,42 +38,84 @@ public void doesNotContainFullStopSpaceBetweenNames_fullStopSpaceNoLetters_retur
     }
 
+    @Test
+    public void doesNotContainApostropheSpaceBetweenNames_noNameAfterSpace_returnTrue() {
+        InputNames noNameAfterSpaceNames = new InputNames("O' ", "O' ");
+        assertThat(doesNotContainAbbreviatedNames(noNameAfterSpaceNames)).isTrue();
+    }
+
     @Test
     public void doesNotContainFullStopSpaceBetweenNames_noNameBeforeFullStop_returnTrue() {
         InputNames noNameBeforeFullStopNames = new InputNames(". John", ". John");
         assertThat(doesNotContainAbbreviatedNames(noNameBeforeFullStopNames)).isTrue();
     }
 
+    @Test
+    public void doesNotContainApostropheSpaceBetweenNames_noNameBeforeApostrophe_returnTrue() {
+        InputNames noNameBeforeApostropheNames = new InputNames("' Connor", "' Connor");
+        assertThat(doesNotContainAbbreviatedNames(noNameBeforeApostropheNames)).isTrue();
+    }
+
     @Test
     public void doesNotContainFullStopSpaceBetweenNames_fullStopSpaceFirstName_returnFalse() {
         InputNames namesWithFullStopSpaceFirstName = new InputNames("St. John", "Smith");
         assertThat(doesNotContainAbbreviatedNames(namesWithFullStopSpaceFirstName)).isFalse();
     }
 
+    @Test
+    public void doesNotContainApostropheSpaceBetweenNames_apostropheSpaceFirstName_returnFalse() {
+        InputNames namesWithApostropheSpaceFirstName = new InputNames("O' Connor", "John");
+        assertThat(doesNotContainAbbreviatedNames(namesWithApostropheSpaceFirstName)).isFalse();
+    }
+
     @Test
     public void doesNotContainFullStopSpaceBetweenNames_fullStopSpaceLastName_returnFalse() {
         InputNames namesWithFullStopSpaceLastName = new InputNames("David", "St. John");
         assertThat(doesNotContainAbbreviatedNames(namesWithFullStopSpaceLastName)).isFalse();
     }
 
+    @Test
+    public void doesNotContainApostropheSpaceBetweenNames_apostropheSpaceLastName_returnFalse() {
+        InputNames namesWithApostropheSpaceLastName = new InputNames("David", "O' Connor");
+        assertThat(doesNotContainAbbreviatedNames(namesWithApostropheSpaceLastName)).isFalse();
+    }
+
     @Test
     public void doesNotContainFullStopSpaceBetweenNames_doubleFullStopNotName_returnTrue() {
         InputNames doubleFullStopNames = new InputNames(".. John", ".. John");
         assertThat(doesNotContainAbbreviatedNames(doubleFullStopNames)).isTrue();
     }
 
+    @Test
+    public void doesNotContainApostropheSpaceBetweenNames_doubleApostropheNotName_returnTrue() {
+        InputNames doubleApostropheNames = new InputNames("'' Connor", "'' Connor");
+        assertThat(doesNotContainAbbreviatedNames(doubleApostropheNames)).isTrue();
+    }
+
     @Test
     public void doesNotContainFullStopSpaceBetweenNames_fullStopAfterSpaceNotName_returnTrue() {
-        InputNames fullStopAfterSpaceNames = new InputNames("St. .", "St. .");
+        InputNames fullStopAfterSpaceNames = new InputNames("St. .","St. .");
```

Comment:
I think you've accidentally changed this line but the previous version had the correct spacing between arguments.

Reply:
Resolved.